benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/workload/users/UserValuesYamlProtocol.scala

```diff
@@ -111,10 +137,26 @@ object ExplorationValuesIntYamlProtocol extends DefaultYamlProtocol {
   }
 
   private def generateStepList(range: List[Int], step: Int, prefix: String): Try[List[Int]] = prefix match {
-    case "+" => RangeWithStep.stepList(range, step, (a: Int, b: Int) => a + b)
-    case "-" => RangeWithStep.stepList(range, step, (a: Int, b: Int) => a - b)
-    case "*" => RangeWithStep.stepList(range, step, (a: Int, b: Int) => a * b)
-    case "^" => RangeWithStep.stepList(range, step, (a: Int, b: Int) => Math.pow(a, b).intValue)
-    case _ => Failure(new BenchFlowDeserializationException("step function not supported", new Throwable))
+    case "+" =>
+      if (step == 0) {
+        Failure(new BenchFlowDeserializationException("step function cannot be identity element: +0", new Throwable))
+      } else {
+        RangeWithStep.stepList(range, step, (a: Int, b: Int) => a + b)
+      }
+
+    case "*" =>
+      if (step == 1) {
```

Comment:
Also `*0` is not valid because you are keep getting `0` (https://en.wikipedia.org/wiki/Absorbing_element). Place also a reference to the way you perform these checks.